Src/Fido2.Models/CredentialCreateOptions.cs

```diff
@@ -96,7 +103,7 @@ public static CredentialCreateOptions Create(Fido2Configuration config, byte[] c
             },
             AuthenticatorSelection = authenticatorSelection,
             Attestation = attestationConveyancePreference,
-            ExcludeCredentials = excludeCredentials ?? new List<PublicKeyCredentialDescriptor>(),
+            ExcludeCredentials = excludeCredentials ?? Array.Empty<PublicKeyCredentialDescriptor>(),

```

Comment:
Same here actually, spec says this is optional: https://www.w3.org/TR/webauthn-3/#dom-publickeycredentialcreationoptions-excludecredentials

Reply:
The spec does say ` defaulting to []`, which would fit the general meaning of "null == no information, empty list == we know that no things exist". Were it not for that part in the spec though, I would have no preference either way.

Reply:
@Regenhardt The way I read the spec, it just means that we don't need to set it. If not set, the browser will default it to be an empty array that is passed to the Authenticator. Not that the RP (we) is required to set it to a empty array.

Reply:
Agree with you, the default value is not necessary it. I removed it and tested manually the change.